### PR TITLE
BaSP-MR-211 Trainer view setup - profile and shedule

### PR DESCRIPTION
**Pull request by EmaBaldor**
Trainer View Setup.
Create a Trainer folder in Components.
Create the Trainer private routes.
Create a Trainer view in /Views.
Modify the layout to display the Trainer view when the role is "TRAINER".
Create profile.
Create Schedule classes.

### CODE REVIEW

**Comment by EmaBaldor** [comment]
[[Trainer] Trainer View Setup](https://trello.com/c/Bj804FuB/211-trainer-trainer-view-setup)

**Comment by EmaBaldor** [comment]
[[Trainer] - Create profile](https://trello.com/c/a0rT0Q4e/218-trainer-create-profile)

**Comment by EmaBaldor** [comment]
[[Trainer] - Schedule](https://trello.com/c/H4R2E2k2/219-trainer-schedule)

File: src/Components/Trainer/Schedule/Modal/modalShedule.module.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  text-align: center;
+  font-weight: bold;
+}
+
+.trainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 5%;
+  font-size: 1.3rem;;
```
Comment (AleMac17) [inline src/Components/Trainer/Schedule/Modal/modalShedule.module.css]: Ema, look this, double ;

**Review by AleMac17 — CHANGES_REQUESTED** [review]

**Review by AleMac17 — APPROVED** [review]

**Review by fernandofernandezuy — APPROVED** [review]
Great job Ema!

**Comment by araefedg** [comment]
Great work Ema!! 😀😀😀

**Pull request merged**